packages/core/src/cli/tsc.ts

```diff
@@ -0,0 +1,19 @@
+import { Abort } from './error'
+import { exec } from './system'
+import { findUp, dirname } from './path'
+import { fileURLToPath } from 'url'
+
+export const TSCNotFoundError = new Abort(
```

Comment:
Nitpick: I'd turn this one into function, that way we don't take memory unnecessarily:

```ts
export const TSCNotFoundError = () => {
  return new Abort(....)
}
```